File: src/main/java/com/mallang/auth/exception/NoAuthenticationSessionException.java
package com.mallang.auth.exception;

import static org.springframework.http.HttpStatus.UNAUTHORIZED;

import com.mallang.common.execption.ErrorCode;
import com.mallang.common.execption.MallangLogException;

public class NoAuthenticationSessionException extends MallangLogException {

    public NoAuthenticationSessionException() {
        super(new ErrorCode(UNAUTHORIZED, "인증 정보가 없거나 만료되었습니다."));
    }
}

File: src/main/kotlin/com/mallang/auth/exception/NoAuthenticationSessionException.kt
package com.mallang.auth.exception

import com.mallang.common.execption.ErrorCode
import com.mallang.common.execption.MallangLogException
import org.springframework.http.HttpStatus.UNAUTHORIZED

class NoAuthenticationSessionException : MallangLogException(
        ErrorCode(UNAUTHORIZED, "인증 정보가 없거나 만료되었습니다.")
)
